pkg/util/BUILD

```diff
@@ -47,9 +47,3 @@ filegroup(
     tags = ["automanaged"],
 )
 
-sh_test(
```

Comment:
https://github.com/kubernetes/kubernetes/pull/99486#issuecomment-786462748 / https://github.com/kubernetes/kubernetes/compare/1b91957eaba55540a355ae69376965acd55474a4..5c1b5e22704f78080fac81dfbe9001ab97be06e6 (you can see that prow handles this now)